woodwork/type_sys/type_system.py

```diff
@@ -93,7 +93,7 @@
 
 DEFAULT_TYPE = Unknown
 
-INFERENCE_SAMPLE_SIZE = 100000
+INFERENCE_SAMPLE_SIZE = 100_000
```

Comment:
We might have to keep this at 100,000. Reducing this exposes issues with larger datasets like `zillow` that has a column which gets inferred as `IntegerNullable` but actually has a float in one of its >90,000 observations. Attempting to transform this with `IntegerNullable` throws an error.